it/uk/gov/hmrc/soletraderidentificationfrontend/stubs/SoleTraderIdentificationStub.scala

```diff
@@ -221,20 +221,20 @@ trait SoleTraderIdentificationStub extends WireMockMethods {
       body = JsString(body)
     )
 
-  def stubRetrieveOverseasTaxIdentifiers(journeyId: String)(status: Int, body: JsValue = Json.obj()): StubMapping =
+  def stubRetrieveOverseasTaxIdentifier(journeyId: String)(status: Int, overseasTaxIdentifier: String = ""): StubMapping =
```

Comment:
You may need the old version until the transition is complete.

Reply:
done